protocol/optyfi-protocol/vaults.md

```diff
@@ -20,9 +20,9 @@ LP Vaults can accept Single-Asset LP Tokens like cDAI or aUSDC as well as Multi-
 
 There will be a separate vault for each underlying asset and risk profile combination. 
 
-Upon launch \(ETA July 2021\), the following vaults will be deployed. 
+Upon launch, the following vaults will be deployed: 
```

Comment:
Change:
Upon *Alpha* launch,....

Reply:
It would probably be more correc to change this to launch? Or beta launch maybe? 
For now, I've changed it to 'During the alpha phase'.